src/AddTodoForm.js

```diff
@@ -1,21 +1,28 @@
-import React from "react";
+import React, { useState } from "react";
 
-let AddTodoForm = (props) => {
-  
-    let handleAddTodo = (event) => {
-        event.preventDefault();
-        const todoTitle = event.target.title.value;
-        console.log(todoTitle);
-        event.target.reset();
-
-        props.onAddTodo(todoTitle );
+let AddTodoForm = ({onAddTodo}) => {
+    const [todoTitle,setTodoTitle] = useState('');
+    let handleTitleChange = (event) => {
+        const todoNewTodoTitle = event.target.value;
+        setTodoTitle(todoNewTodoTitle);
     }
+
+    const handleAddTodo = (e) => {
+        e.preventDefault();
+        onAddTodo(
+            {
+                title: todoTitle,
+                id: Date.now()
+            }
+        );
+        setTodoTitle(''); 
+    };
   
   return (
         <form onSubmit={handleAddTodo}>
              <label htmlFor="todoTitle">Title</label>
              <br/>
-            <input type="text" name="title" id="todoTitle" />
+            <input type="text" name="title" value={onAddTodo.todoNewTodoTitle} id="todoTitle" onChange={handleTitleChange} />
```

Comment:
You're doing a great job setting the todo state on line 18 but I believe the input doesn't clear up after submitting a todo because the value should be **todoList** instead of **onAddTodo.todoNewTodoTitle** - something like this:

`value={todoList}`